### DISCUSSION TITLE
安装之后资源管理器无限重启

### DISCUSSION THREAD

**Discussion by LZR7106** [zetaloop/ExplorerPatcher#11] [General]
不管是最新的英文版还是最新的中文版，安装之后，资源管理器开始无限重启，科学的不科学网络连接都试过了，但是并没有用，重启电脑也不行
win11 家庭版，版本号10.0.22000.778

**Comment by LQZ77** [comment]
无限重启+无法打开开始菜单

**Comment by zetaloop** [comment]
好家伙，可能是不兼容版本，英文版不支持的话我暂无处理方法
控制台日志里有没有下载失败之类的？

**Comment by LWkeeptrying** [comment]
![image](https://github.com/zetaloop/ExplorerPatcher/assets/133618023/75e6ca43-d709-478a-8982-28e29ab8518b)
一直用旧版本的，今天更新了Windows以后就出现无限重启资源管理器的情况，直到最后安装了目前最新的explorerpatcher才好了
![image](https://github.com/zetaloop/ExplorerPatcher/assets/133618023/3ff9c68b-16a5-4cfb-bd3d-6dab19eee1c7)

**Comment by peoplechinapower** [comment]
我也遇到了相同的情况，不过现在秒数直接显示在时间里了，不知道是不是bug

**Reply by peoplechinapower** [reply]
不是bug 是Windows加的新功能 可以看秒

**Comment by xuan4730** [comment]
两个中文版本的都装了一下，都还是英文，什么地方姿势不对？

**Reply by zetaloop** [reply]
你是不是win10呀？win10的部分之前还没汉化的，你安装现在最新的pre-release试试？

**Comment by xuan4730** [comment]
谢谢回复，我已经使用上了，很棒的程式，如果能把系统文件夹也变成圆角而不仅仅是开始菜单就更加完美了，最后还要感谢你们的辛勤创作与创造。
________________________________
发件人: zetaloop ***@***.***>
发送时间: 2023年10月17日 12:53
收件人: zetaloop/ExplorerPatcher ***@***.***>
抄送: xuan4730 ***@***.***>; Comment ***@***.***>
主题: Re: [zetaloop/ExplorerPatcher] 安装之后资源管理器无限重启 (Discussion #11)


你是不是win10呀？win10的部分之前还没汉化的，你安装现在最新的pre-release试试？

―
